styles.css

```diff
@@ -523,59 +506,399 @@ form li + li {
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    padding: 64px 24px;
+    position: absolute;
+    top: 614px;
+    gap: 30px;
+    width: 1850px;
+  }
+
+  .recent-works {
+    width: 500px;
+    margin-top: 345px;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    justify-content: center;
+  }
+
+  .disabled {
+    margin-bottom: 130px;
+    margin-top: 600px;
+  }
+
+  .fa-horizontal-rule,
+  #separator {
+    display: none;
+  }
+
+  .project-container {
+    display: grid;
+    grid-template-columns: repeat(3, 369px);
+    grid-template-rows: repeat(2, 474px);
+    justify-content: center;
+    gap: 1rem;
+    width: 100%;
+    border-radius: 8px;
+  }
+
+  .project1,
+  .project2,
+  .project3,
+  .project4,
+  .project5,
+  .project6 {
+    border: 1px solid #d0d9d4;
+    border-radius: 1%;
+  }
+
+  .projectimages {
+    width: 367px;
+    height: 252px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    width: 1850px;
+    margin-top: 1600px;
+    background-image: url(images/aboutmeimage.png);
+    background-size: cover;
+    background-repeat: repeat-x;
+  }
+
+  .after-resume {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    gap: 30px;
+    width: 100%;
+  }
+
+  .aboutme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    width: 400px;
+    height: 189px;
+    margin-bottom: 80px;
+  }
+
+  .titleaboutme {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 900px;
+    font-size: 40px;
+    line-height: 40px;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .aboutmeparagraph {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    width: 770px;
+    height: 109px;
+    text-align: center;
+    color: #42526e;
+  }
+
+  .projectbuttonresume {
+    align-items: center;
+    font-family: 'Inter', sans-serif;
+    letter-spacing: 0.05em;
+    padding: 12px;
+    gap: 10px;
+    width: 200px;
+    height: 50px;
+    bottom: 15px;
+    font-weight: 550px;
+    font-size: 20px;
+    position: relative;
+    border: none;
+    background: #36b37f;
+    border-radius: 4px;
+    color: #fff;
+    cursor: pointer;
+  }
+
+  .languages,
+  .frameworks,
+  .skills {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    background: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    gap: 20px;
+    width: 349px;
+    height: 352px;
+    justify-content: center;
+  }
+
+  .contactandform {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .contactinfo {
```

Comment:
:red_circle: As requested by the previous reviewer, Kindly apply the missing background on this section to match the template. You can follow this example to extract the icons of this section from the Figma Template :arrow_down: 
![Peek 2023-01-26 23-15](https://user-images.githubusercontent.com/26736582/214955609-1d12ea78-a10b-461a-bf4b-530c1aa62724.gif)